src/lib.rs

```diff
@@ -1,3 +1,71 @@
+//! # A minimal RPC library for use with [iroh].
```

Comment:
would be nice to have this description in the readme.md and then include the readme in here, maybe?